apps/preview/nuxt/pages/showcases/MegaMenu/MegaMenuNavigation.vue

```diff
@@ -76,7 +76,7 @@
               :key="activeMenu.key"
               ref="megaMenuRef"
               :style="style"
-              class="hidden md:grid gap-x-6 grid-cols-4 bg-white shadow-lg p-6 left-0 right-0 outline-none"
+              class="hidden md:grid gap-x-6 grid-cols-4 bg-white shadow-lg p-6 left-0 right-0 outline-none z-10"
```

Comment:
I're aware of this issue, and we're trying to find a way to fix it for all blocks. We'd rather not include just one fix right now if you don't mind. You can take part in the discussion [here](https://vsf-company.slack.com/archives/C02UKV884RW/p1685620283674229).

Reply:
No problem, thank you for updating me